src/GenAPI/Microsoft.DotNet.GenAPI.Tool/Program.cs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+
+namespace Microsoft.DotNet.GenAPI.Tool
+{
+    /// <summary>
+    /// CLI frontend for the Roslyn-based GenAPI.
+    /// </summary>
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            // Global options
+            Option<string[]> assembliesOption = new("--assemblies",
+                description: "The path to one or more assemblies or directories with assemblies.",
+                parseArgument: ParseAssemblyArgument)
+            {
+                AllowMultipleArgumentsPerToken = true,
+                Arity = ArgumentArity.OneOrMore,
+                IsRequired = true
+            };
+
+            Option<bool> resolveAssemblyReferencesOption = new("--resolve-assembly-references",
+                "Default is `false`. If true, tries to resolve assembly reference.");
+
+            Option<string[]?> assemblyReferencesOption = new("--assembly-references",
+                description: "Paths to assembly references or their underlying directories for a specific target framework in the package.",
+                parseArgument: ParseAssemblyArgument)
+            {
+                AllowMultipleArgumentsPerToken = true,
+                Arity = ArgumentArity.ZeroOrMore
+            };
+
+            Option<string[]?> excludeAttributesFilesOption = new("--exclude-attributes-files",
```

Comment:
Singular option name:

```suggestion
            Option<string[]?> excludeAttributesFilesOption = new("--exclude-attributes-file",
```